3p-ecosystem/src/main/java/com/google/homesampleapp/screens/home/HomeFragment.kt

```diff
@@ -409,16 +444,56 @@ class HomeFragment : Fragment() {
     }
   }
 
+  // -----------------------------------------------------------------------------------------------
+  // Composables
+
+  @Composable
+  private fun HomeScreen(homeViewModel: HomeViewModel) {
```

Comment:
The `HomeScreen` composable should take only lambdas and state if possible.

An example can be seen in the [interests screen](https://github.com/android/nowinandroid/blob/main/feature/interests/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/interests/InterestsScreen.kt#L38) in "Now in Android".

It's seems a verbose, but this makes it possible to preview the Composable without having to create a `HomeViewModel` instance, and also avoids some nuance about ViewModel instances being seeing as [unstable](https://developer.android.com/jetpack/compose/performance/stability) by the compose compiler.

Reply:
done